<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #232526;
 border-top-left-radius:15px;
border-top-right-radius:15px;
border-bottom-left-radius:15px;
border-bottom-right-radius:15px;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="jahan">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>891</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../New folder (2)/e.jpg</pixmap>
    </property>
   </widget>
   <widget class="QFrame" name="zarb">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: #302b63;
background-color: rgb(0, 0, 49);
background-color: rgb(0, 0, 53);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>0</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Traditional Arabic</family>
       <pointsize>29</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>World Clock</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="iran">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../New folder (2)/Iran-Flag-icon.png</normaloff>../New folder (2)/Iran-Flag-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="amrica">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../New folder (2)/United-States-of-America-Flag-icon.png</normaloff>../New folder (2)/United-States-of-America-Flag-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="canada">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../New folder (2)/Canada-Flag-icon.png</normaloff>../New folder (2)/Canada-Flag-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ustralia">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>310</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../New folder (2)/Australia-Flag-icon.png</normaloff>../New folder (2)/Australia-Flag-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="brazil">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../New folder (2)/Brazil-Flag-icon.png</normaloff>../New folder (2)/Brazil-Flag-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="russia">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../New folder (2)/Russia-Flag-icon.png</normaloff>../New folder (2)/Russia-Flag-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="indaia">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../New folder (2)/india.png</normaloff>../New folder (2)/india.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../New folder (2)/Turkey-Flag-icon.png</normaloff>../New folder (2)/Turkey-Flag-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="japan">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>210</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../New folder (2)/Japan-Flag-icon.png</normaloff>../New folder (2)/Japan-Flag-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="china">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../New folder (2)/China-Flag-icon.png</normaloff>../New folder (2)/China-Flag-icon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="hour">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Traditional Arabic</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFAFBD;
background-color:none;
</string>
    </property>
    <property name="text">
     <string>06:21:57</string>
    </property>
   </widget>
   <widget class="QLabel" name="hour_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Traditional Arabic</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFAFBD;
background-color:none;
</string>
    </property>
    <property name="text">
     <string>06:21:57</string>
    </property>
   </widget>
   <widget class="QLabel" name="hour_8">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>80</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Traditional Arabic</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFAFBD;
background-color:none;
</string>
    </property>
    <property name="text">
     <string>06:21:57</string>
    </property>
   </widget>
   <widget class="QLabel" name="hour_9">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Traditional Arabic</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFAFBD;
background-color:none;
</string>
    </property>
    <property name="text">
     <string>06:21:57</string>
    </property>
   </widget>
   <widget class="QLabel" name="hour_10">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Traditional Arabic</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFAFBD;
background-color:none;
</string>
    </property>
    <property name="text">
     <string>06:21:57</string>
    </property>
   </widget>
   <widget class="QLabel" name="hour_11">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>200</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Traditional Arabic</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFAFBD;
background-color:none;
</string>
    </property>
    <property name="text">
     <string>06:21:57</string>
    </property>
   </widget>
   <widget class="QLabel" name="hour_12">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Traditional Arabic</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFAFBD;

background-color:none;
</string>
    </property>
    <property name="text">
     <string>06:21:57</string>
    </property>
   </widget>
   <widget class="QLabel" name="hour_13">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Traditional Arabic</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFAFBD;
background-color:none;
</string>
    </property>
    <property name="text">
     <string>06:21:57</string>
    </property>
   </widget>
   <widget class="QLabel" name="hour_14">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Traditional Arabic</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFAFBD;
background-color:none;
</string>
    </property>
    <property name="text">
     <string>06:21:57</string>
    </property>
   </widget>
   <widget class="QLabel" name="hour_15">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>300</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Traditional Arabic</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FFAFBD;
background-color:none;
</string>
    </property>
    <property name="text">
     <string>06:21:57</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
